tests/unit/createOrUpdateStagingDeployTest.ts

```diff
@@ -53,7 +55,7 @@ beforeAll(() => {
             },
         },
         paginate: jest.fn().mockImplementation((objectMethod: () => Promise<{data: unknown}>) => objectMethod().then(({data}) => data)),
-    } as typeof GithubUtils.octokit;
+    } as unknown as typeof GithubUtils.internalOctokit;
```

Comment:
Maybe we can export `InternalOctokit` and use it here

Reply:
Done!